lib/id003/id003.js

```diff
@@ -175,6 +175,7 @@ Id003.prototype._run = function _run (cb) {
       }
     }, 500)
   })
+  this.initialized = true
```

Comment:
BillValidator isn't initialized yet, this should be set inside the callbacks after the validator gives us the denominations.

@littlenaut can you explain why you set this in parallel to opening the rs-232 port? This kind of thing can cause a lot of problems in nodeJS programming.

Reply:
@joshmh You're right in should be placed on the line above.

Reply:
We shouldn't be sending commands until we're finished fully initializing the peripheral, otherwise those commands could get in the way of a successful initialization. The more important point is to be careful about following NodeJS's callback-based execution and not setting flags in parallel because that leads to a lot of subtle bugs that can be hard to catch and debug.